include/Metabox/Posts/Category.php

```diff
@@ -90,14 +90,16 @@ public function move( $options ) {
 		) );
 
 		foreach ( $posts as $post ) {
-			$current_cats = array_diff( wp_get_post_categories( $post->ID ), array( $options['old_cat'] ) );
+			$current_cats = wp_get_post_categories( $post->ID );
```

Comment:
`wp_get_post_categories` can retrun `WP_Error`. We need to add an additional check.

_I think this was an existing issue and not related to your change_.

Reply:
> `wp_get_post_categories` can retrun `WP_Error`. We need to add an additional check.

Fixed.